### PR TITLE
Make repair a method on EDS

### LINKED ISSUE #77: api: rework `RepairExtendedSquare` to be `Repair` method on `ExtendedDataSquare`
This way, lib users can avoid reimporting the square repeatedly for cases where Repair has to be called multiple times.

### PR DESCRIPTION
**Pull request by adlerjohn**
Fixes #77

### CODE REVIEW

File: extendeddatacrossword.go [hunk 1]
```diff
@@ -52,29 +49,13 @@ func (e *ErrByzantineCol) Error() string {
 // prior to the Byzantine row or column being repaired, and the Byzantine row
 // or column prior to repair is returned in the error with missing shares as
 // nil.
-func RepairExtendedDataSquare(
+func (eds *ExtendedDataSquare) RepairExtendedDataSquare(
```
Comment (Wondertan) [inline extendeddatacrossword.go:52]: Let's name it just ‘Repair’. The struct itself already mentions that it is an extended data sqaure

**Comment by vgonkivs** [comment]
LGTM. Agree with @Wondertan that `Repair` is enough.

**Review by vgonkivs — APPROVED** [review]

File: extendeddatacrossword.go (hunk 1 above)
Comment (adlerjohn) [reply extendeddatacrossword.go]: Fixed in [54b8499](https://github.com/celestiaorg/rsmt2d/pull/79/commits/54b8499ca6ab761069e30a7b79b4025035984db4)

**Review by Wondertan — APPROVED** [review]

**Review by liamsi — APPROVED** [review]
👌🏼

**Pull request merged**